modules/contact-form/grunion-contact-form.php

```diff
@@ -1028,9 +1028,32 @@ static function parse( $attributes, $content ) {
 		return $r;
 	}
 
+	/**
+	 * Returns a success message to be returned if the form is sent via AJAX.
+	 *
+	 * @param Integer $feedback_id
+	 * @param Grunion_Contact_Form $form
+	 *
+	 * @return String $message
+	 */
 	static function success_message( $feedback_id, $form ) {
-		$r_success_message = '';
+		return wp_kses(
+			"<blockquote>\n"
```

Comment:
Are we leaving the blockquote and br's in here?  I'd propose that we add a class to the blockquote and style it with extra space below rather than using BRs, unless there's a reason not to.  I'm just concerned about our users' abilities to modify the design without having to hack JP